PyZAPI/pyzapi/routes.py

```diff
@@ -0,0 +1,10 @@
+def includeme(config):
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+
+    # relevant routes for the blog post
```

Comment:
agreed.  Let's pull this in and then come up with a list of items to iterate forward on.  Be great to get a few more hands on it in this state.